### DISCUSSION TITLE
master-detail form

### DISCUSSION THREAD

**Discussion by magicommike** [Budibase/budibase#3472] [Help] [answered]
I am trying to create an orders management system (for practice and to understand how budibase works).
I create tables (Suppliers, Customers, Products, OrdersMaster, OrderDetails)
I have a problem; I can’t create a master-detail form.
When a user selects an order (from table “OrdersMaster”, I want to display all relative rows from the table “OrderDetails”  table.

Can somebody help me?

Regards

![Screen-1](https://user-images.githubusercontent.com/63601674/142831707-2ac48f15-9f50-46a2-95d3-ddf5a8808599.png)

**Comment by mjashanks** [comment, marked as answer]
Hi There - this is written up here: https://docs.budibase.com/guides/master-detail (click on the screenshot in there to enlarge)

**Reply by NicolasFabre81** [reply]
@mjashanks this link is dead

**Comment by magicommike** [comment]
Thank you,

**Comment by GiuseppePiscopoHypex** [comment]
Hi. Has that doc page moved since then? The posted link seems broken. Thanks!
